build.gradle

```diff
@@ -23,6 +23,11 @@ plugins {
 
 repositories {
   jcenter()
+  repositories {
```

Comment:
Is it really needed?

Reply:
The `com.google.cloud.tools:com.google.cloud.tools.jib.gradle.plugin` seems only present in [mavenCentral](https://mvnrepository.com/artifact/com.google.cloud.tools.jib/com.google.cloud.tools.jib.gradle.plugin?repo=gradle-plugins) and not jcenter. 

The Gradle plugin portal is a mirror to jcenter and Maven Central thats why it works withe this snippet.

The options I can see:
* Ask Google to publish the `jib-gradle-plugin` to jcenter (the [jib maven plugin](https://bintray.com/bintray/jcenter/com.google.cloud.tools%3Ajib-maven-plugin) is already published). Do you know anyone able to do that?
* Wait for the official publication, that will freeze this PR for a while
* Update the project build files to include `mavenCentral()` (cleaner than pointing the plugin portal) and documentation to explicit the need of Maven Central as a replacement/complement of JCenter.

WDYT?